### PR TITLE
fix(google): prevent parent server group from overwriting null clone …

### PR DESCRIPTION
**Pull request by maggieneterval**
…autohealing policies

Corresponding deck PR: https://github.com/spinnaker/deck/pull/6337

### CODE REVIEW

File: clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/CopyLastGoogleServerGroupAtomicOperation.groovy [hunk 1]
```diff
@@ -260,7 +260,8 @@ class CopyLastGoogleServerGroupAtomicOperation extends GoogleAtomicOperation<Dep
     println(",, description autohealing policy ${description.autoHealingPolicy}")
     println(",, ancestor autohealing policy ${ancestorAutoscalingPolicy}")
     println(",, ancestor autohealing policy description ${ancestorAutoscalingPolicyDescription}")
-    newDescription.autoHealingPolicy = description.autoHealingPolicy ?: ancestorAutoHealingPolicyDescription
+    newDescription.autoHealingPolicy = (description.autoHealingPolicy != null || description.overwriteAncestorAutoHealingPolicy) ?
```
Comment (jtk54) [inline clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/CopyLastGoogleServerGroupAtomicOperation.groovy]: Remove the `println`s. I think I snuck those in there.

Comment (maggieneterval) [reply clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/CopyLastGoogleServerGroupAtomicOperation.groovy]: :+1:

**Comment by maggieneterval** [comment]
@spinnakerbot cherry-pick 1.10

**Comment by maggieneterval** [comment]
@spinnakerbot cherry-pick 1.11

**Pull request merged**